src/main/java/br/com/quintoandar/quintolog/entity/enums/SecurityQuestion.java

```diff
@@ -0,0 +1,25 @@
+package br.com.quintoandar.quintolog.entity.enums;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.stream.Stream;
+
+@Getter
+@AllArgsConstructor
+public enum SecurityQuestion {
+
+    TEACHER(1, "logError.question1"),
```

Comment:
por que nesse enum tem números e nos outros só as strings? Se não for necessário, eu diria pra retirar. Se for, eu diria pra colocar nos outros.